comps/comps-foreman-rhel7.xml

```diff
@@ -360,6 +360,7 @@
       <packagereq type="default">katello-selinux</packagereq>
       <packagereq type="default">keycloak-httpd-client-install</packagereq>
       <packagereq type="default">mod_passenger</packagereq>
+      <packagereq type="default">nodejs-node-gyp</packagereq>
```

Comment:
Was this handled by nodejs-node-gyp PR we merged ?

Reply:
I don't think it was, which... is problematic now that I think of it.